### DISCUSSION TITLE
Locations in EME are listed in Google - deactivation possible?

### DISCUSSION THREAD

**Discussion by t-b-x** [liedekef/events-made-easy#358] [General]
Hello, 
we use EME as event calendar for our own events and also for external events to list and information. The locations of these events can be found by Google and are linked to our homepages' location site. This may work as designed, but is missinterpred by the external users. If they found our homepage as first link if they search for keyword+location, they assume we are originated to that external location and/or external event. Can we prevent Google to index the location site?

Example: search google "karate karthause"
It will show our homepage karate-porz.de as first entry. But we are not originated to that location.

Thanks in advance
Norman

**Comment by liedekef** [comment]
I do not give google search support ... the events can be linked to a location and that location can be anything you want. But since the location info is defined on your site, google will use that info. And search results depend on who/where: if I look for "karate karthause" on google, it points me to https://www.karate-porz.de/veranstaltungsort/29/sporthalle-gymnasium-karthause/ , which seems pretty accurate to me.

**Reply by t-b-x** [reply]
Thanks for the feedback. I understand, that it works as designed and the search result is correct. But the context of the link is missleading at this point: it was a external event we added to our calendar for reference. As it is now found by Google, people think we do events there, which is not true. We have no relation to that location (and other locations). 
So question is, if we can ommit, that EME is creating location sites? 
Is a possible workaround to delete/disable the fixed location list and just use manual entries in the events (this option exists already)?

Thanks & kind regards
Norman

**Reply by liedekef** [reply]
The problem is that you added something to your site for someone else. If it is an external url, you can enter that in EME, but if you enter actual location info, the event/location will show that.
Either don't enter location info for that event or don't enter that event (for now).
There's also the option to use the filter eme_extra_event_headers_json_filter , which allows you to change the event headers used by google, see https://www.e-dynamics.be/wordpress/hooks-and-filters/

**Reply by t-b-x** [reply]
Thanks for the feedback. We will adapt how external events are entered. 
Kind regards, Norman
